### ISSUE TITLE
Issue : BZminer - Radiant - Connection Drop - Zombie shares

### ISSUE THREAD

**Issue by Kizabin** [bzminer/bzminer#206]
Hi everyone, 

Just here to report  a problem ive been having with 14.1.1 ... its dropping the connection after failing to report a share after a period of time ... the pool says that its zombie shares that is happening ... vipor.net ... just happening with my one machine running 6 3080tis ... other system is fine 3 3080ti and 6 3070 ... was recomended by the pool to switch to 12.2 ... everything seems to be fine ... so far

**Comment by zerjiok** [comment]
yeah I have the same issue some 3060 and 3060ti, also I notices with the same OCs using latest version 14.1.1 radiant hashrate dropped, so I went back to 12.2

**Comment by bzminer** [comment]
Hey guys, thanks for reporting this issue, I'll try to take care of it in the next release
